new-lamassu-admin/src/pages/Triggers/helper.js

```diff
@@ -546,6 +568,12 @@ const Requirement = () => {
     (!values.requirement?.suspensionDays ||
       values.requirement?.suspensionDays < 0)
 
+  const hasCustomRequirementError =
+    !!errors.requirement &&
+    !!touched.requirement?.customInfoRequestId &&
+    (!values.requirement?.customInfoRequestId ||
+      !R.isNil(values.requirement?.customInfoRequestId))
```

Comment:
What do you think about export this and use it on Wizard.js instead of redefining it